hhvm-serve.go

```diff
@@ -1,10 +1,10 @@
 package main
 
-import "github.com/beberlei/hhvm-serve/fcgiclient"
+import "github.com/LionsAd/hhvm-serve/fcgiclient"
```

Comment:
We could indeed, we would need to remove the added new method and move to the hhvm-serve itself, but that is the only change I made compared to upstream IIRC.